data/human/hails.txt

```diff
@@ -2708,6 +2708,167 @@ phrase "friendly civilian"
 		"neodymium and neodymium" "neodymium"
 		"neodymium or neodymium" "neodymium"
 
+phrase "friendly civilian"
+	word
+		"Be careful of"
+		"A warning about"
+		"Be suspicious of"
+		"Don't trust"
+	word
+		" those space casinos. I blew "
+	word
+		"half a"
+		"a"
+		"two"
+		"three"
+		"four"
+		"five"
+	word
+		" million credits on "
+	word
+		"Ada"
+		"Calda"
+		"Farseer"
+		"Geminus"
+		"Glory"
+		"Hestia"
+		"Mainsail"
+		"Martini"
+		"Pearl"
+		"Vail"
+		"Vinci"
+	word
+		" "
+	word
+		"blackjack."
+		"poker."
+		"backgammon."
+		"baccarat."
+		"roulette."
+
+phrase "friendly civilian"
+	word
+		"Have you read the latest edition of "
+	word
+		"Cosmic Chronicles"
+		"Daring Discoveries"
+		"Norn Nebula News"
+		"Southern Rim Spacer News"
+		"Stars and Starships"
+		"Sunracer Stellar Spectacles"
+		"Tarazed Today"
+		"the Galactic Gazette"
+		"the New Moonshake"
+		"the Satellite Sentinel"
+	word
+		"? "
+	word
+		"It says"
+		"It claims"
+	word
+		" that "
+		" "
+	word
+		"pulse showers are good for your quantum pores"
+		"exposing yourself to neodymium increases your risk of spacepox"
+		"shoddily grown ion algae exhausts your psychic karma"
+		"laser acupuncture lets you communicate with nebula ghosts"
+		"star crystals bring luck to long distance relationships"
+		"consuming moon dust accelerates hair growth but causes lunacy"
+		"excessive exposure to zero gravity can lead to phantom limb syndrome"
+		"alien prosthetics often come with unexpected telepathic abilities"
+		"stardate tattoos increase your chances of alien abduction"
+	word
+		"."
+		"!"
+
+phrase "friendly civilian"
+	word
+		"I'm no weapons expert, but it feels better firing "
+	word
+		"energy blasters"
+		"particle cannons"
+		"proton guns"
```

Comment:
References to the specific outfits should be capitalised.
```suggestion
		"Energy Blasters"
		"Particle Cannons"
		"Proton Guns"
```
Also, I'm curious as to why Plasma Cannons are not included in this list?
I think they're much more similar to Paritcle Cannons and Proton Guns than Energy Blasters.
```suggestion
		"Energy Blasters"
		"Particle Cannons"
		"Plasma Cannons"
		"Proton Guns"
```